### ISSUE TITLE
Not able to import UMAPVisualizer from yellowbrick.text

### ISSUE THREAD

**Issue by soniaarora** [DistrictDataLabs/yellowbrick#727]
I downloaded the latest package of yellowbrick but it was file of umap_vis.py was missing from the package of yellowbrick.text

**Comment by soniaarora** [comment]
I downloaded the latest package of yellowbrick but it was file of umap_vis.py was missing from the package of yellowbrick.text

**Comment by bbengfort** [comment]
Hi @soniaarora the UMAP visualizer hasn't been released yet; it's part of our v1.0 release which we're expecting to conclude in the next month or two -- I'm very sorry for the inconvenience. If you'd like to use the latest features you can install Yellowbrick directly from GitHub as follows:

```
$ pip install git+https://github.com/DistrictDataLabs/yellowbrick.git@develop
```

This should install our develop branch and make sure you also reference our develop documentation: https://www.scikit-yb.org/en/develop/. Let me know if this works!

**Comment by bbengfort** [comment]
Hi, @soniaarora I hope my suggestion helped, let me know if you need any further assistance, and please stay tuned for the v1.0 release!
